core/serial_command.py

```diff
@@ -532,12 +485,12 @@ def select_channels(self, channels):
         """Select multiple channels, expects list for channels"""
         self.logger.debug("Select channels %s %s" % (channels, type(channels)))
         self.clear_channel()
-        command = _cmd_channel_select_many_start
+        command = p._cmd_channel_select_many_start
         for channel in channels:
             print channel
             command += chr(channel)
-        command += _cmd_channel_select_many_end
-        buffer_check = "B"+str((int(channels[0])-1)/8+1)+_cmd_channel_select_many_end
+        command += p._cmd_channel_select_many_end
+        buffer_check = p._cmd_disable_ext_trig+str((int(channels[0])-1)/8+1)+_cmd_channel_select_many_end
```

Comment:
Correct, well spotted. No idea why this didn't crash, maybe the function wasn't being called.